cmd/metal-api/internal/service/machine-service.go

```diff
@@ -399,26 +399,26 @@ func (r machineResource) webService() *restful.WebService {
 		Returns(http.StatusOK, "OK", v1.MachineResponse{}).
 		DefaultReturns("Error", httperrors.HTTPErrorResponse{}))
 
-	ws.Route(ws.POST("/upload/bios").
+	ws.Route(ws.POST("/upload/bios/{vendor}/{board}/{revision}").
```

Comment:
I would rather combine these endpoint with bmc and create one firmware-service where one upload endpoint exists which contains bmc/bios/networkcard firmware blobs.

WDYT ?

Reply:
Sounds good, I'll do so after my tests